tests/functional/spec/features/hub_and_spoke/hub_and_spoke_custom_content.spec.js

```diff
@@ -0,0 +1,28 @@
+const HouseholdSummary = require('../../../generated_pages/hub_and_spoke_custom_content/household-section-summary.page.js');
+const HowManyPeopleLiveHere = require('../../../generated_pages/hub_and_spoke_custom_content/how-many-people-live-here.page.js');
+const DoesAnyoneLiveHere = require('../../../generated_pages/hub_and_spoke_custom_content/does-anyone-live-here.page.js');
+
+const HubPage = require('../../../base_pages/hub.page.js');
+
+
+describe('Feature: Hub and Spoke with custom content', function () {
+  const hub_and_spoke_schema = 'test_hub_and_spoke_custom_content.json';
+
+  it('When the questionnaire is incomplete, then custom content should be displayed correctly', function () {
+    browser.openQuestionnaire(hub_and_spoke_schema);
+    expect($(HubPage.displayedGuidance()).getText()).to.contain('Guidance displayed on hub when incomplete');
```

Comment:
Given that you are testing the title as part of the complete state, I think the title and submission button should also be tested in the incomplete state to verify that they do in fact have the expected default text.